benchmarks/shelley3pools/prepare_genesis-generic.sh

```diff
@@ -0,0 +1,182 @@
+#!/usr/bin/env bash
```

Comment:
why is this file "generic"? how does it differ from the normal version?

Reply:
We had hard-coded genesis and utxo key numbers. This takes it from the `parameters` file now.